packages/helm-charts/common/values.yaml

```diff
@@ -1 +1,33 @@
 imagePullPolicy: IfNotPresent
+
+# Full-node variables
+geth:
+  image:
+    repository: us.gcr.io/celo-testnet/geth
+    tag: 1b40b25d315bfcd792138e288ea61351d6c44d09
+    imagePullPolicy: IfNotPresent
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: "500m"
+    limits:
+      memory: "512Mi"
+      cpu: "1000m"
+  debug: false
+  ping_ip_from_packet: false
+  in_memory_discovery_table: false
+  syncmode: full
+  gcmode: full
+  verbosity: 2
+  public_ips:
+  - 52.149.51.48
```

Comment:
Where does this IP come from?

Reply:
These are the static ips created in Azure. The `values.yaml` file in the chart root serves as default value for the chart variables if no values are provided. In this case, the `geth.public_ips` variable is overwritten by celotool, and the static ips are created in the function `createStaticIPs` in packages/celotool/src/lib/aks-fullnode.ts. I will update these ips to make more clear that these ips have to be replaced to some ips that belongs to the same Resource Group of your aks nodes.